rust/operator-binary/src/superset_controller.rs

```diff
@@ -660,11 +671,19 @@ fn build_server_rolegroup_statefulset(
     pb.add_container(metrics_container);
 
     if merged_config.logging.enable_vector_agent {
+        let resources = ResourceRequirementsBuilder::new()
+            .with_cpu_request("250m")
+            .with_cpu_limit("500m")
+            .with_memory_request("128Mi")
+            .with_memory_limit("128Mi")
+            .build();
+
         pb.add_container(product_logging::framework::vector_container(
             resolved_product_image,
             CONFIG_VOLUME_NAME,
             LOG_VOLUME_NAME,
             merged_config.logging.containers.get(&Container::Vector),
+            resources,
```

Comment:
Nit: I would prefer to directly inline the `resources` here

Reply:
Inlined in 87a973d

Reply:
I have the feeling you didn't inline it but fine for me ^^

Reply:
Yes one builder wasn't inlined. Since 98953db every `ResourceRequirementsBuilder` is inlined as requested.